pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -118,13 +118,16 @@ Listen Sockets
     ``enableRenegotiation``, ``exactPathMatching``, ``maxConcurrentTCPConnections`` and ``releaseBuffers`` options added.
     ``internalPipeBufferSize`` now defaults to 1048576 on Linux.
 
+  .. versionchanged:: 1.8.0
```

Comment:
should this be 1.7.0?

Reply:
@rgacogne was talking to target 1.8.0 https://github.com/PowerDNS/pdns/pull/11027#discussion_r755165497

Reply:
I'd like to finally get 1.7.0 out of the door, but that means I need to stop adding things to it ;)